routes/add_event.js

```diff
@@ -8,15 +8,23 @@ module.exports = (req, res) => {
     categories: req.body.categories,
     accessibilityOptions: req.body.accessibilityOptions,
     startTime: req.body.startDate + 'T' + req.body.startTime,
-    endTime: req.body.endDate + 'T' + req.body.endTime,
+    endTime: req.body.startDate + 'T' + req.body.endTime,
     cost: req.body.cost,
-    imageUrl: req.body.imageUrl,
-    en: {
-      name: req.body.name,
-      description: req.body.description
-    }
+    imageUrl: req.body.imageUrl
   };
 
+  if (req.body.name_english) {
```

Comment:
Need to handle english and arabic descriptions too

Reply:
but descriptions are not required but ill add them if they exist with a ternary operator